src/js/components/pageTitle.js

```diff
@@ -0,0 +1,18 @@
+const pageTitle = data => {
```

Comment:
What is this file added in this PR? It was already develop in a previous one

Reply:
Please perform a `git pull origin develop` from your branch to have the latest changes

Reply:
Done!!!